src/components/MainContent/MainContent.tsx

```diff
@@ -29,13 +29,15 @@ const MainContent = () => {
 
       <TransferPanel />
 
-      {mergedTransactions?.length > 0 && (
+      {mergedTransactionsToShow?.length > 0 && (
         <>
-          <TransactionsTable transactions={mergedTransactions?.slice(0, 5)} />
+          <TransactionsTable
+            transactions={mergedTransactionsToShow?.slice(0, 5)}
```

Comment:
For the # of txns to show by default? What's the advantage of the making it an ENV var?

Reply:
the main advantage is first having it as a variable so that we reuse on the other places where 5 is hardcoded. But I agree the ENV part is not really necessary